File: roommate-python/Roommate.py
class Roommate:
    def __init__(self, name, age, personality, hobby):
        self.name = name
        self.age = age
        self.personality = personality
        self.hobby = hobby

    def display_info(self):
        print("Roommate Details:")
        print(f"Name: {self.name}")
        print(f"Age: {self.age}")
        print(f"Personality: {self.personality}")
        print(f"Hobby: {self.hobby}")

r = Roommate("Brian", 20, "Calm and organized", "Watching football")
r.display_info()

File: roommate-cpp/Roommate.cpp
#include <iostream>
#include <string>

class Roommate {
public:
    std::string name;
    int age;
    std::string personality;
    std::string hobby;

    Roommate(std::string nm, int ag, std::string pers, std::string hob)
        : name(nm), age(ag), personality(pers), hobby(hob) {}

    void display_info() const {
        std::cout << "Roommate Details:\n";
        std::cout << "Name: " << name << std::endl;
        std::cout << "Age: " << age << std::endl;
        std::cout << "Personality: " << personality << std::endl;
        std::cout << "Hobby: " << hobby << std::endl;
    }
};

int main() {
    Roommate r("Brian", 20, "Calm and organized", "Watching football");
    r.display_info();
    return 0;
}